### ISSUE TITLE
R Basics: Introduction

### ISSUE THREAD

**Issue by rosemm** [arcus/education_modules#83]
I was copying over content from this module to update the ggplot2 module and I noticed a few little errors: 

- [x] line 186: missing alt text
- [x] line 240: reference to wrong folder for code
- [x] line 255: reference to wrong folder for code
- [x] line 262: reference is to branch (and its the visualize branch as well, not the intro one), so I think it will break if/when that branch is deleted, although it's still working now
- [x] the png and gif referenced in line 262 appear to not be in the media folder on main
- [x] line 276: I think the last file name should be `transform_exercises.rmd` rather than `transform.rmd`
- [x] line 671: reference to wrong folder for code

**Comment by pm0kjp** [comment]
TYSM!  On it!

**Comment by franzenr** [comment]
@pm0kjp @rosemm Rose, thanks so much for being thorough on this! I definitely was not nearly as vigilant on the branch references as I should have been! 

I want to clarify that the alt text for line 186 should have a blank tag (it looks like it has a  totally blank attribute, which is also incorrect, thanks for catching that!) (ie, it should be ![""]) 

All of the information contained in that image is also in the accompanying text, and therefore the image should be treated as "decorative", as it does not add any additional information, and having an alt text read aloud leads to unnecessary redundancies for folks using screen readers.

**Comment by pm0kjp** [comment]
> @pm0kjp @rosemm Rose, thanks so much for being thorough on this! I definitely was not nearly as vigilant on the branch references as I should have been (in fact, I realized in our meeting that I need to go back through all the QAs I did and check on the gif paths in particular, as all of those slipped under my radar).
> 
> I want to clarify that the alt text for line 186 should have a blank tag (it looks like it has a totally blank attribute, which is also incorrect, thanks for catching that!) (ie, it should be ![""])
> 
> All of the information contained in that image is also in the accompanying text, and therefore the image should be treated as "decorative", as it does not add any additional information, and having an alt text read aloud leads to unnecessary redundancies for folks using screen readers.

notes!  Will add the "" instead of reiterating.  Good catches and this was super dense so I don't feel bad, and I hope you don't either @franzenr
